README.md

```diff
@@ -1,5 +1,6 @@
 # Go SDK for EdgeCast CDN
-This is the official Go SDK for EdgeCast. It contains services that interact with the EdgeCast APIs.
+This is the official Go SDK for EdgeCast. It contains services that interact 
+with the EdgeCast APIs.
```

Comment:
The official Go SDK for interacting with EdgeCast APIs.

Reply:
updated